.teamcity/pluginData/settings.xml

```diff
@@ -0,0 +1,23 @@
+<settings xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/SETTINGS/1.0.0" xsi:schemalocation="http://maven.apache.org/SETTINGS/1.0.0
```

Comment:
Not sure how exactly this works. I hope @guylabs knows.

Reply:
What are you referring to ?

Reply:
How the TeamCity DSL know about this file references via `userSettingsSelection`

Reply:
That is some TC Maven plugin magic see https://www.jetbrains.com/help/teamcity/maven-server-side-settings.html#MavenServer-SideSettings-MavenSettingsResolutionontheServerSide